### DISCUSSION TITLE
back to snipe-IT pre flight after changing password

### DISCUSSION THREAD

**Discussion by ictyves** [snipe/snipe-it#10516] [Q&A] [answered]
I set up snipe-IT an got it working. When i was getting ready to start, i forgot my password. 
first i did this in mysql:
SET PASSWORD FOR 'snipeit'@'localhost' = PASSWORD('newpassword');
When i tried to log in then i could give in username and password in the frontend of snipeIT but got to the pre flight screen with this error:
 _D'oh! Looks like we can't connect to your database. Please update your database settings in your .env file. Your database says: SQLSTATE[HY000] [1045] Access denied for user 'snipeit'@'localhost' (using password: YES) (SQL: select 2 + 2)__

I went to /var/www/html/snipe and did sudo nano .env
there i changed the password and saved it 'ctrl+o'
when i go to the frontend, i can't login anymore and go directly to http://serverip/setup with same database error.
Can someone please assist me in what i can do? I know the name of the database, of the user and also password.

Thx in advance,

**Comment by snipe** [comment]
You changed your MySQL database password instead of your Snipe-IT user password. Change it back to whatever is in your .env file under DB_PASSWORD, or change your DB_PASSWORD to whatever you made your newpassword be.

**Reply by ictyves** [reply]
Hello There, yes that is what i thought. So i changed my sql pw back to the original and than changed my snipe-it pw to that same password.
But after that i can't even log in to snipe frontend anymore. It just goes directly to the pre flight screen with everything green accept the database.
Could it have something to do with rights to files?

**Reply by snipe** [reply]
It shouldn't - and if database is not green, then you still haven't fixed the database connection issue. The two things that decide whether to show you the setup screen is the [middleware](https://github.com/snipe/snipe-it/blob/master/app/Http/Middleware/CheckForSetup.php) and the `setupCompleted()` Settings model method:

https://github.com/snipe/snipe-it/blob/be7e6ed84794bd2d5d3dc4f5b43523d00bc8f8cd/app/Models/Setting.php#L114-L127

So it still looks like your database credentials are wrong.

**Comment by ictyves** [comment]
It seems weird. I have a screenshot with the correct password but when i
change my .env file and also my Snipe-IT password to that password, the
problem doesn't get solved.

Is there a way to see what is going wrong or to reset the database password
in a clean way so i can use my snipe-IT environment again without
reinstalling?

Thx in advance!


Op vr 14 jan. 2022 om 20:15 schreef snipe ***@***.***>:

**Comment by ictyves** [comment, marked as answer]
Just managed to solve it. I reset my SnipeIT pw again and got back to the login-screen.
After that i created a new admin user with commandline. Logged in with that user and then solved other users problem.
Thx for the help!
